nameable.rb

```diff
@@ -0,0 +1,7 @@
+class Nameable
+  def correct_name(name)
+    corrected_name = name.capitalize
+    corrected_name = name[0...10] if name.length > 10
+    corrected_name
+  end
+end
```

Comment:
- [ ] According to the instructions, this **Nameable** class should return a descriptive `NotImplementedError`. Kindly fix that.